client/src/lib/attempt-result.js

```diff
@@ -414,29 +420,29 @@ export function attemptResultsWarning(
         return attempt > 0 && centiseconds / attempted < 30 * 100;
       });
       if (lowTimeIndex !== -1) {
-        return `
-        The result you're trying to submit seems to be impossible:
-        attempt ${lowTimeIndex + 1} is done in
-        less than 30 seconds per cube tried.
-        If you want to enter minutes, don't forget to add two zeros
-        for centiseconds at the end of the score.
-      `;
+        return {
+          description: `The result you're trying to submit seems to be impossible:
+            attempt ${lowTimeIndex + 1} is done in
+            less than 30 seconds per cube tried.
+            If you want to enter minutes, don't forget to add two zeros
+            for centiseconds at the end of the score.`
+        };
       }
     } else {
       const worstSingle = Math.max(...completeAttempts);
       const inconsistent = worstSingle > bestSingle * 4;
       if (inconsistent) {
-        return `
-          The result you're trying to submit seem to be inconsistent.
-          There's a big difference between the best single
-          (${formatAttemptResult(bestSingle, eventId)}) and the worst single
-          (${formatAttemptResult(worstSingle, eventId)}).
-          Please check that the results are accurate.
-        `;
+        return {
+          description: `The result you're trying to submit seem to be inconsistent.
+            There's a big difference between the best single
+            (${formatAttemptResult(bestSingle, eventId)}) and the worst single
+            (${formatAttemptResult(worstSingle, eventId)}).
+            Please check that the results are accurate.`
+        };
       }
     }
   }
-  return null;
+  return {};
```

Comment:
I think returning an empty object is not meaningful, so I would prefer to go with the approach I originally suggested :)

Reply:
Destructuring could still be done `const {...} = attemptResultsWarning(...) || {}`, though I wouldn't do that either, since `if (submissionWarning)` is more clear.